contrib/examples/cluster-deployment-template.yaml

```diff
@@ -84,6 +84,8 @@ objects:
   metadata:
     name: ${CLUSTER_NAME}
     namespace: ${CLUSTER_NAMESPACE}
+    annotations:
+      clusteroperator.openshift.io/delete-after: 5m
```

Comment:
Why so low? Right now it takes at least 20m for the cluster to finish provisioning

Reply:
My mistake. I'm going to probably remove this in the default template, it'll bite someone no matter what we put in there.